std_msgs/UInt8 op
std_msgs/UInt8 comm_interface
std_msgs/UInt16 period
std_msgs/String sys_dst
